app/models/show.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Show < ApplicationRecord
```

Comment:
you added `, null: false` in the migration for some fields, add those validations here, `presence: true`